jekyll/_docs/ios-builds-on-os-x.md

```diff
@@ -442,18 +443,18 @@ machine:
 
 test:
   override:
-    - xctool
-      -reporter pretty
-      -reporter junit:$CIRCLE_TEST_REPORTS/xcode/results.xml
-      -reporter plain:$CIRCLE_ARTIFACTS/xctool.log
-      CODE_SIGNING_REQUIRED=NO # no need to do code signing at this point
-      CODE_SIGN_IDENTITY=      # as it will be done in the deployment step
-      PROVISIONING_PROFILE=
-      -destination 'platform=iOS Simulator,name=iPhone 6,OS=latest'
-      -sdk iphonesimulator
-      -workspace MyWorkspace.xcworkspace # your workspace name
-      -scheme "My Scheme" # your scheme name
-      build build-tests run-tests
+    - set -o pipefail &&
+      xcodebuild
+        CODE_SIGNING_REQUIRED=NO # no need to do code signing at this point
```

Comment:
i'm sad to say i dont know how the yml parser works - but just a check here as it appears to have a space in front of CODE_SIGNING_REQUIRED  - which the original text did not